src/server/plugins/engine/components/EmailAddressField.ts

```diff
@@ -28,15 +28,8 @@ export class EmailAddressField extends FormComponent {
   }
 
   getViewModel(formData: FormData, errors?: FormSubmissionErrors) {
-    const schema = this.schema
     const viewModel = super.getViewModel(formData, errors)
 
-    if ('max' in schema && schema.max) {
-      viewModel.attributes = {
-        maxlength: schema.max
```

Comment:
Can I double check where this change came from?

If we've removed features from fields we should do the same in the types/schema

https://github.com/DEFRA/forms-designer/blob/880d443c0a029be66a56da8193e2ee5279fd0737/model/src/components/types.ts#L23-L52

Reply:
The property `max` is not getting removed from the field, it's still there and used to build the field's joi schema.

These removed lines just stop `maxlength` being passed as an attribute to the GDS Text Field njk macro, stopping:

`<input type="text" maxlength="10">`

Reply:
Thanks for clarifying this Dave, it wasn't obvious to me either. I can see in the constructor that max is set on the componentSchema.